src/components/ButtonLink/styles.ts

```diff
@@ -11,7 +11,7 @@ export const ButtonLink = styled(Link, {
   shouldForwardProp: (prop) => prop !== 'primary',
 })<LinkProps & ButtonLinkProps>`
   display: inline-block;
-  padding: 7px 24px;
+  padding: 11px 24px;
```

Comment:
Any reason for these values not to be in `rem` btw?

Reply:
No, good catch, it's fixed!